tests/conftest.py

```diff
@@ -22,6 +24,12 @@ def notify_api():
     app = NotifyApiFlaskApp("test")
     create_app(app)
 
+    # Attach some routes that can be used as helpers for specific tests.
+    test_blueprint.before_request(requires_no_auth)
+    app.register_blueprint(test_blueprint, url_prefix="/test")
+    admin_test_blueprint.before_request(requires_admin_auth)
+    app.register_blueprint(admin_test_blueprint, url_prefix="/admin-test")
```

Comment:
i'm a little uncertain about these blueprints - i think i'd rather they were added just for a specific test rather than everything (also the names should probably more closely reflect the `before_request` auth functions being the interesting thing?)

but not sure if there's a nice way to do that in a separate fixture that we can just define in test_logging_filters or similar? 🤔

Reply:
yeah, for testing error handlers i created a new app entirely and then attached error handlers and had a route for each test, but that might not be quite as applicable/readable/useful for these tests 🤔  

https://github.com/alphagov/notifications-api/blob/ee50423c8c7386e0f11d3e682acc7d04cac3d092/tests/app/v2/test_errors.py#L15-L28

Reply:
Ah hmm yeah. I would rather test the real application here I think 🤔